shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -115,6 +115,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     result([self clipboardHasStrings]);
   } else if ([method isEqualToString:@"LiveText.isLiveTextInputAvailable"]) {
     result(@([self isLiveTextInputAvailable]));
+  } else if ([method isEqualToString:@"LookUp.initiate"]) {
```

Comment:
I think it's a good idea to keep this method separate from text input like this :+1:

Reply:
So although we can check if a term exists, on native iOS the look up screen gets shown regardless if the term exists or not - it'll just say like "Definition not found" or something. I left it out so we can replicate that behaviour as well.
Also Im responding here because for some reason I can't reply to your comment directly haha

Reply:
what does `initiate` mean? are there any other operations after initiation?